data/aurora/VTBB.lartcc

```diff
@@ -0,0 +1,1117 @@
+T;VTBB_S_TMA;N013.04.30.351;E100.25.01.996;
+T;VTBB_S_TMA;N013.49.18.558;E100.24.14.988;
+T;VTBB_S_TMA;N013.53.36.800;E100.35.46.300;
+//VTBB_SW_TMA
+T;VTBB_SW_TMA;N013.49.18.558;E100.24.14.988;
+T;VTBB_SW_TMA;N013.04.30.351;E100.25.01.996;
+T;VTBB_SW_TMA;N013.04.41.579;E100.24.10.534;
+T;VTBB_SW_TMA;N013.04.53.668;E100.23.19.276;
+T;VTBB_SW_TMA;N013.05.06.614;E100.22.28.234;
+T;VTBB_SW_TMA;N013.05.20.413;E100.21.37.425;
+T;VTBB_SW_TMA;N013.05.35.062;E100.20.46.864;
+T;VTBB_SW_TMA;N013.05.50.556;E100.19.56.564;
+T;VTBB_SW_TMA;N013.06.06.891;E100.19.06.541;
+T;VTBB_SW_TMA;N013.06.24.061;E100.18.16.809;
+T;VTBB_SW_TMA;N013.06.42.062;E100.17.27.383;
+T;VTBB_SW_TMA;N013.07.00.889;E100.16.38.278;
+T;VTBB_SW_TMA;N013.07.20.536;E100.15.49.506;
+T;VTBB_SW_TMA;N013.07.40.997;E100.15.01.084;
+T;VTBB_SW_TMA;N013.08.02.267;E100.14.13.024;
+T;VTBB_SW_TMA;N013.08.24.338;E100.13.25.342;
+T;VTBB_SW_TMA;N013.08.47.206;E100.12.38.050;
+T;VTBB_SW_TMA;N013.09.10.862;E100.11.51.162;
+T;VTBB_SW_TMA;N013.09.35.301;E100.11.04.693;
+T;VTBB_SW_TMA;N013.10.00.515;E100.10.18.656;
+T;VTBB_SW_TMA;N013.10.26.496;E100.09.33.065;
+T;VTBB_SW_TMA;N013.10.53.238;E100.08.47.932;
+T;VTBB_SW_TMA;N013.11.20.732;E100.08.03.271;
+T;VTBB_SW_TMA;N013.11.48.970;E100.07.19.094;
+T;VTBB_SW_TMA;N013.12.17.945;E100.06.35.416;
+T;VTBB_SW_TMA;N013.12.47.647;E100.05.52.249;
+T;VTBB_SW_TMA;N013.13.18.068;E100.05.09.604;
+T;VTBB_SW_TMA;N013.13.49.199;E100.04.27.496;
+T;VTBB_SW_TMA;N013.14.21.031;E100.03.45.936;
+T;VTBB_SW_TMA;N013.14.53.554;E100.03.04.936;
+T;VTBB_SW_TMA;N013.15.26.760;E100.02.24.508;
+T;VTBB_SW_TMA;N013.16.00.639;E100.01.44.665;
+T;VTBB_SW_TMA;N013.16.35.180;E100.01.05.418;
+T;VTBB_SW_TMA;N013.17.10.373;E100.00.26.778;
+T;VTBB_SW_TMA;N013.17.46.209;E099.59.48.757;
+T;VTBB_SW_TMA;N013.18.22.677;E099.59.11.366;
+T;VTBB_SW_TMA;N013.18.59.766;E099.58.34.615;
+T;VTBB_SW_TMA;N013.19.37.465;E099.57.58.517;
+T;VTBB_SW_TMA;N013.20.15.763;E099.57.23.081;
+T;VTBB_SW_TMA;N013.20.54.650;E099.56.48.318;
+T;VTBB_SW_TMA;N013.21.34.113;E099.56.14.238;
+T;VTBB_SW_TMA;N013.22.14.141;E099.55.40.851;
+T;VTBB_SW_TMA;N013.22.54.723;E099.55.08.167;
+T;VTBB_SW_TMA;N013.23.35.847;E099.54.36.195;
+T;VTBB_SW_TMA;N013.24.17.500;E099.54.04.946;
+T;VTBB_SW_TMA;N013.24.59.671;E099.53.34.428;
+T;VTBB_SW_TMA;N013.25.42.347;E099.53.04.651;
+T;VTBB_SW_TMA;N013.26.25.515;E099.52.35.622;
+T;VTBB_SW_TMA;N013.27.09.164;E099.52.07.351;
+T;VTBB_SW_TMA;N013.27.53.280;E099.51.39.847;
+T;VTBB_SW_TMA;N013.28.37.850;E099.51.13.116;
+T;VTBB_SW_TMA;N013.29.22.861;E099.50.47.168;
+T;VTBB_SW_TMA;N013.30.08.301;E099.50.22.010;
+T;VTBB_SW_TMA;N013.30.54.155;E099.49.57.650;
+T;VTBB_SW_TMA;N013.31.40.411;E099.49.34.094;
+T;VTBB_SW_TMA;N013.32.27.054;E099.49.11.349;
+T;VTBB_SW_TMA;N013.33.14.072;E099.48.49.423;
+T;VTBB_SW_TMA;N013.34.01.450;E099.48.28.323;
+T;VTBB_SW_TMA;N013.34.49.175;E099.48.08.053;
+T;VTBB_SW_TMA;N013.35.37.232;E099.47.48.621;
+T;VTBB_SW_TMA;N013.49.18.558;E100.24.14.988;

```

Comment:
Same as above.

Reply:
P' Tonmai has answer the issue here, use the VTBB_APP in the IVAOTH database